zulipterminal/model.py

```diff
@@ -1164,10 +1164,13 @@ def _handle_update_message_flags_event(self, event: Event) -> None:
                       self.controller, -1)
 
     def formatted_local_time(self, timestamp: int,
-                             *, show_seconds: bool) -> str:
+                             *, show_seconds: bool = False,
+                             show_year: bool = False) -> str:
         local_time = datetime.datetime.fromtimestamp(timestamp)
         if show_seconds:
             return local_time.strftime('%a %b %d %H:%M:%S')
+        if show_year:
+            return local_time.strftime('%a %b %-d %Y %-H:%M:%S')
```

Comment:
This makes `show_seconds` True by default?
We may have a `format_date` like string variable, in which we could append supported formats as per query?

Reply:
That's a good idea... I'll refactor it😅

Reply:
I suggested a similar approach in #917, which we'll need to integrate with this at some point in any case.